css/index.css

```diff
@@ -3,144 +3,395 @@
-    width: 800px;
-    margin: 0 auto;
+  width: 800px;
+  margin: 0 auto;
 }
 
 .top-content {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
+}
+.header-content {
+  width: 100%;
+  display: flex;
+  justify-content: stretch;
+}
+nav {
+  width: 100%;
+  margin: 0 0 1em 0;
+  padding: 0;
+  display: flex;
+  justify-content: stretch;
+  height: 100%;
+  width: 150%;
+}
+
+.logo img {
+  width: 100%;
+}
+
+.links {
+  width: 60%;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+nav a {
+  color: black;
+  text-decoration: none;
+}
+
+.p-container {
+  width: 100%;
+
+  padding-bottom: 20px;
+  margin-left: 3%;
+  margin-top: 2%;
 }
 
 .top-content .text-container {
-    width: 48%;
-    padding: 0 1%;
-    padding-bottom: 20px;  
+  width: 48%;
+  padding: 0 1%;
+  padding-bottom: 20px;
+  margin-top: 2%;
 }
 
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
+}
+
+.middle-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-content: space-between;
+  width: 50%;
+
+  padding-bottom: 20px;
+  padding-left: 0;
+}
+.middle-container h2 {
+  padding-top: 4%;
+}
+
+.middle2 {
+  padding: 5%;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
+}
+
+.middle1 {
+  padding: 5%;
+  margin-bottom: 20px;
+  border-bottom: 1px dashed black;
+}
+
+.content2 {
+  padding-bottom: 5%;
+  margin-bottom: 2%;
+  border-bottom: 1px dashed black;
+}
+
+.content21 {
+  padding-top: 10%;
 }
 
+.content12 {
+  padding-top: 10%;
+}
+
+.content1 {
+  padding-bottom: 5%;
+  margin-bottom: 2%;
+  border-bottom: 1px dashed black;
+}
+
+.bottom {
+  margin-left: 3%;
+}
+.content3 {
+  width: 100%;
+  padding-bottom: 20px;
+}
+
+button {
+  margin: 3%;
+  padding: 5px;
+  border: 2px solid black;
+  border-radius: 10px;
+  background-color: white;
+  font-size: 18px;
+}
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+  padding: 0 2%;
+  margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: stretch;
+  margin: 5%;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 100%;
+  height: 100px;
+  background: black;
+  margin: 20px 2.5%;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-.bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+.boxes {
```

Comment:
It looks like you have added some CSS, great work!  You can also shorten this up a little bit since all your boxes padding and margin are the same by doing .box1 .box2 .box3 .box4 { padding: 6% 4%; margin: 3% 3% }.